src/library-authoring/library-access/data/api.js

```diff
@@ -0,0 +1,68 @@
+import { ensureConfig, getConfig } from '@edx/frontend-platform';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+ensureConfig([
+  'STUDIO_BASE_URL',
+], 'Library API service');
+
+function setupRequest() {
+  return { client: getAuthenticatedHttpClient(), baseUrl: getConfig().STUDIO_BASE_URL };
+}
+
+/* eslint-disable-next-line import/prefer-default-export */
+export async function fetchUsers(libraryId) {
+  const { client, baseUrl } = setupRequest();
+  const response = await client.get(
+    `${baseUrl}/api/libraries/v2/${libraryId}/team/`,
+  ).catch((error) => {
+    /* Normalize error data. */
+    const apiError = Object.create(error);
+    apiError.message = null;
+    apiError.fields = JSON.parse(error.customAttributes.httpErrorResponseData);
+    throw apiError;
+  });
+  return response.data;
+}
+
+export async function addUserToLibrary({ libraryId, data }) {
+  const { client, baseUrl } = setupRequest();
+  const response = await client.post(
+    `${baseUrl}/api/libraries/v2/${libraryId}/team/`,
+    data,
+  ).catch((error) => {
+    /* Normalize error data. */
+    const apiError = Object.create(error);
+    apiError.message = null;
+    apiError.fields = JSON.parse(error.customAttributes.httpErrorResponseData);
+    throw apiError;
+  });
+  return response.data;
+}
+
+export async function setUserAccessLevel({ libraryId, user, level }) {
```

Comment:
Nit: I don't like the name `user` as a parameter, especially in plain JS with no type information. Is it a user object? an ID? an email? Using `userId` or `userObj` would make it more clear. Though as mentioned in the other PR, we'll likely need to change to just `username`.